libs/langchain/langchain/llms/clarifai.py

```diff
@@ -23,27 +27,21 @@ class Clarifai(LLM):
         .. code-block:: python
 
             from langchain.llms import Clarifai
-            clarifai_llm = Clarifai(pat=CLARIFAI_PAT, \
-                user_id=USER_ID, app_id=APP_ID, model_id=MODEL_ID)
+            clarifai_llm = Clarifai(pat=CLARIFAI_PAT,user_id=USER_ID, app_id=APP_ID, model_id=MODEL_ID)
```

Comment:
```suggestion
            clarifai_llm = Clarifai(user_id=USER_ID, app_id=APP_ID, model_id=MODEL_ID)
```
I guess now don't allow pat as arg, so we can remove this?